pkgs/by-name/no/nordvpn/package.nix

```
@@ -0,0 +1,175 @@
+            patchPathFtn = (
+              fname: "${tunnelblickSrc}/third_party/sources/openvpn/openvpn-${old.version}/patches/${fname}"
+            );
+          in
+          (old.patches or [ ])
+          ++ [
+            (patchPathFtn "02-tunnelblick-openvpn_xorpatch-a.diff")
+            (patchPathFtn "03-tunnelblick-openvpn_xorpatch-b.diff")
+            (patchPathFtn "04-tunnelblick-openvpn_xorpatch-c.diff")
+            (patchPathFtn "05-tunnelblick-openvpn_xorpatch-d.diff")
+            (patchPathFtn "06-tunnelblick-openvpn_xorpatch-e.diff")
+          ];
+      });
+    })
+  ];
+
+in
+buildGoModule (finalAttrs: {
+
+  pname = "nordvpn";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    hash = "sha256-0GgMIFi6qrO7NG94vvWprwWr+8j87M5eH5W/pCtSWqs=";
+    owner = "NordSecurity";
+    repo = "nordvpn-linux";
+    tag = finalAttrs.version;
+  };
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    makeWrapper
+    pkg-config
+  ];
+
+  buildInputs = [
+    # cgo build dependencies go here
+    # https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/go.section.md#envcgo_enabled-var-go-cgo_enabled
+    # libxml2 2.14.[0-4] breaks daemon
+    libxml2_13
+  ];
+
+  vendorHash = "sha256-GREoxjXqb28nabNvvcGjQA1rG4h9e/gINqEPY4++6Oo=";
+
+  modPostBuild = ''
+    patch -p0 < ${./gokogiri-xpath-expression.patch}
+  '';
+
+  preBuild = ''
+    # use path $out/bin instead of /usr/lib
+    sed -i "s|/usr/lib/nordvpn|$out/bin|" internal/constants.go
+
+    # use path <<openvpnPatch>>/bin/openvpn instead of /usr/lib/nordvpn/openvpn
+    old_ovpn_path='filepath.Join(internal.AppDataPathStatic, "openvpn")'
+    new_ovpn_path=\"${patchedPkgs.openvpn}/bin/openvpn\"
+    sed -i "s|$old_ovpn_path|$new_ovpn_path|" daemon/vpn/openvpn/config.go
+  '';
+
+  ldflags = [
+    "-X main.Environment=prod"
+    "-X main.Hash=${finalAttrs.version}"
+    # changing the salt would result in nordvpnd crash for existing users
+    "-X main.Salt=f1nd1ngn3m0"
+    "-X main.Version=${finalAttrs.version}"
+  ];
+
+  subPackages = [
+    "cmd/cli"
+    "cmd/daemon"
+    "cmd/norduser"
+  ];
+
+  doCheck = true;
+
+  checkPhase = ''
+    runHook preCheck
+
+    go test ./cli
+    # skip tests that require network access
+    go test ./daemon -skip 'TestTransports|TestH1Transport_RoundTrip'
+    go test ./norduser
+
+    runHook postCheck
+  '';
+
+  postInstall = ''
+    # rename to standard names
+    BIN_DIR=$out/bin
+    install $BIN_DIR/cli $BIN_DIR/nordvpn
+    install $BIN_DIR/daemon $BIN_DIR/nordvpnd
+    install $BIN_DIR/norduser $BIN_DIR/norduserd
+    rm $BIN_DIR/{cli,daemon,norduser}
+
+    # nordvpn needs icons for the system tray and notifications
+    ASSETS_PATH=$out/share/icons/hicolor/scalable/apps
+    install -D assets/icon.svg $ASSETS_PATH/nordvpn.svg
+    for file in assets/*; do
+      install "$file" "$ASSETS_PATH/nordvpn-$(basename $file)"
+    done
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/nordvpnd --set PATH ${
```

Shouldn't this be
```suggestion
    wrapProgram $out/bin/nordvpnd --prefix PATH ${
```
?